frontend/model/contracts/chatroom.js

```diff
@@ -40,10 +40,8 @@ export const messageType: any = objectMaybeOf({
     type: unionOf(...Object.values(MESSAGE_NOTIFICATIONS).map(v => literalOf(v))),
     params: mapOf(string, string) // { username }
   }),
-  replyingMessage: objectOf({
+  replying: objectOf({
```

Comment:
Also in `mesage` object, we have `id` and `text` fields so `replyingMessage` makes sense here too.
@taoeffect, what do you think?

Reply:
I didn't catch the distinction. So maybe replyingMessage in ChatMain.vue should be renamed to replyingMessageText since it is plain text. Although I'm not sure because that name sounds a bit like it was about the text of the reply rather than the text of the original message. Good names are sometimes hard to find.

Reply:
@Silver-IT is this what we're replying to? If so what about `replyingTo` or `replyTo`?

Reply:
`replyingMessage` also seemed fine to me. Or the more descriptive but longer, `replyingToMessage`.

Reply:
Okay. `replyingMessage`!

Reply:
Oh sorry. I didn't implement `message replying` function in this PR. It's in the next PR. #1279